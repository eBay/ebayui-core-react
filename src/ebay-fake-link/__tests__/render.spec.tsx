import React from 'react'
import { composeStories } from '@storybook/react'

import * as stories from './index.stories'
import { render } from '@testing-library/react'

const { Default } = composeStories(stories)

describe('<EbayFakeLink /> rendering', () => {
    it('renders default story correctly', () => {
        const { container } = render(<Default />)
        expect(container).toMatchSnapshot()
    })

    it('renders disabled button correctly', () => {
        const { container } = render(<Default disabled />)
        expect(container).toMatchSnapshot()
    })
})
