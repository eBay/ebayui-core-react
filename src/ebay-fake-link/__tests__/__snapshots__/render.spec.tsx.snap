// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EbayFakeLink /> rendering renders default story correctly 1`] = `
<div>
  <button
    class="fake-link"
    type="button"
  >
    Fake-Link
  </button>
</div>
`;

exports[`<EbayFakeLink /> rendering renders disabled button correctly 1`] = `
<div>
  <button
    class="fake-link"
    disabled=""
    type="button"
  >
    Fake-Link
  </button>
</div>
`;
